### ISSUE TITLE
Record position while no GSM/wifi present

### ISSUE THREAD

**Issue by sivascu** [tananaev/traccar-client-ios#17]
Would it be possible to add in an extra feature like recording the position of the device when GSM/WIFI is not available and send all recorded positions once a connection is possible ? 

This could also be implemented for the android app.

**Comment by tananaev** [comment]
Yes, I plan to add this feature in one of the future releases. Android app already has it.

**Comment by w32gold** [comment]
Hi Anton,
I would to ask if this feature is already available in iOS? 
Thanks!

**Comment by tananaev** [comment]
Yes, it's already available in both Android and iOS apps.

**Comment by w32gold** [comment]
​Great!
Thanks..​

- Sherwin
